SixTrack/sixtrack.s

```diff
@@ -18428,7 +18428,7 @@ cc2008
       write(4,*) ch
       goto 3
  90   continue
-10000 format(a16,1x,i2,1x,d21.15,1x,d21.15,1x,d16.10)
+10000 format(a16,1x,i2,1x,d22.15,1x,d22.15,1x,d17.10)
```

Comment:
I don't know any fort.4 user. Perhaps Frank himself. I would try to simplify if you see the opportunity and wait for complaints....